Sources/TuistLoader/Mappers/ProjectNameAndOrganizationMapper.swift

```diff
@@ -28,6 +28,11 @@ public class ProjectNameAndOrganizationMapper: ProjectMapping {
             project.organizationName = organizationName
         }
 
+        // Xcode project development region
+        if let developmentRegion = developmentRegionOverride() {
```

Comment:
I know that you mention this in the PR description, but I disagree that this should not have a new mapper. I'd say that having a mapper with a really trivial logic is actually an advantage, rather than not - and changing `developmentRegion` does not really fit semantically into this mapper. Or what do you think, @kwridan?

Reply:
+1 Having smaller mappers make them easier to reason about. 

Grouping a few related ones into a single mapper is fine too - but then naming gets funnier `ProjectNameOrganizationDevelopmentRegionMapper` - perhaps a simplification `ProjectAttributesMapper` or something along these lines.

Reply:
I agree here. If we have a mapper specific to this, we can easily search for it and reason about the logic in it.

Reply:
Ok, so I split it to two separate Mappers 👍